### PR TITLE
Changelog 1.11.0

### PR DESCRIPTION
**Pull request by rockstardev**
https://github.com/btcpayserver/btcpayserver/issues?q=is%3Aissue+milestone%3A1.11.0

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+
+### Improvements
+
+* Improved Payment Requests List View (#3872 #5065) @TChukwuleta
+* Improve create first store view (#5008 #5181) @dennisreimann
+* Invoice lists: Show icons for payment methods (#5084 #5137) @dennisreimann
+* Checkout v2: Play sound when invoice is paid (#5085 #5113) @dennisreimann @webwworthy
+* Apps: Add direct file upload in item editor (#5086 #5140) @dennisreimann
+* Form invoice amount adjusters, useful for shipping and other addon amounts to invoice (#5087 #5158) @Kukks @dennisreimann
+* Add product categories to POS apps (#5088 #5078) @NicolasDorier
```
Comment (pavlenex) [inline Changelog.md:25]: I think this should be under features?

Comment (ndeet) [reply Changelog.md:25]: - maybe also the form invoice adjustments with the new compute field for shipping costs? 
- maybe also the Checkout play sounds as feature?

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,29 @@
 # Changelog
 
+## 1.11.0
+
+### New Features
+
+* Complete overhaul of Invoice Reporting (#5095 #5155) @NicolasDorier
```
[the remark this answers is not in the document]
Comment (pavlenex) [reply Changelog.md:7]: https://github.com/btcpayserver/btcpayserver/pull/5199/commits/6d65feca4c2ad3a380c9bcdf4432572212805233

File: Changelog.md (hunk 1 above)
Comment (pavlenex) [reply Changelog.md]: https://github.com/btcpayserver/btcpayserver/pull/5199/commits/6d65feca4c2ad3a380c9bcdf4432572212805233

File: Changelog.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pavlenex) [reply Changelog.md:7]: https://github.com/btcpayserver/btcpayserver/pull/5199/commits/6d65feca4c2ad3a380c9bcdf4432572212805233

File: Changelog.md (hunk 1 above)
Comment (pavlenex) [reply Changelog.md]: https://github.com/btcpayserver/btcpayserver/pull/5199/commits/6d65feca4c2ad3a380c9bcdf4432572212805233

**Pull request merged**